### PR TITLE
#333 Moving dependencies to Dev dependencies

### PR DESCRIPTION
**Pull request by adamdavies1**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Please also include any acceptance criteria if you have any.

- Made some dependencies to dev dependencies

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce
the testing if necessary.

Before PR's can be merged they will need to be tested by QA and approved where
applicable. To flag the change to QA assign **@XGovFormBuilder/qa** as one of the reviewers.

- [ ] Build, run and run smoke tests

# Checklist:

- [ ] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto main and there are no code conflicts

### CODE REVIEW

File: model/package.json [hunk 1]
```diff
@@ -28,40 +28,40 @@
...
     "@babel/core": "^7.11.1",
     "@babel/eslint-parser": "^7.17.0",
+    "@babel/eslint-plugin": "^7.16.5",
     "@babel/plugin-proposal-class-properties": "7.16.7",
     "@babel/plugin-proposal-export-default-from": "^7.16.7",
     "@babel/plugin-proposal-private-methods": "^7.16.11",
     "@babel/plugin-transform-runtime": "^7.17.0",
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-typescript": "^7.16.7",
+    "@babel/runtime": "^7.17.2",
```
Comment (jenbutongit) [inline model/package.json:47]: babel-runtime needs to be a main dep https://babeljs.io/docs/en/babel-runtime#usage

File: runner/package.json [hunk 2]
```diff
@@ -40,35 +40,24 @@
...
-    "@hapi/code": "8.0.1",
     "@hapi/cookie": "^11.0.2",
     "@hapi/crumb": "^8.0.1",
     "@hapi/hapi": "^20.0.3",
     "@hapi/inert": "^6.0.5",
     "@hapi/jwt": "^2.1.0",
-    "@hapi/lab": "24.0.0",
     "@hapi/scooter": "^6.0.1",
-    "@hapi/vision": "^6.1.0",
-    "@hapi/wreck": "^17.1.0",
```
Comment (jenbutongit) [inline runner/package.json:52]: vision and wreck are modules we use in the code.

Vision is used to render the views. Wreck is a HTTP utility. I forget if we use it so just double check that

File: runner/package.json [hunk 3]
```diff
@@ -119,16 +107,28 @@
...
     "babel-plugin-module-name-mapper": "^1.2.0",
     "cheerio": "^1.0.0-rc.10",
+    "chrome-launcher": "^0.13.4",
     "cli-ux": "^6.0.9",
     "code": "5.2.4",
+    "date-fns": "^2.24.0",
     "eslint": "^8.11.0",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-tsdoc": "^0.2.14",
+    "expr-eval": "^2.0.2",
```
Comment (jenbutongit) [inline runner/package.json:121]: this is used as a main dep

File: model/package.json (hunk 1 above)
Comment (adamdavies1) [reply model/package.json]: Moved back to dependencies

File: runner/package.json (hunk 3 above)
Comment (adamdavies1) [reply runner/package.json]: Moved back to dependencies

File: runner/package.json (hunk 2 above)
Comment (adamdavies1) [reply runner/package.json]: removed vision and wreck, and replaced them with the hapi versions

**Review by RichardBray — APPROVED** [review]
This change looks fine to me, codewise. But as this isn't a package that people download and install is there a massive need/benefit for this refactor?

**Comment by RichardBray** [comment]
> > This change looks fine to me, codewise. But as this isn't a package that people download and install is there a massive need/benefit for this refactor?
> 
> Hi @RichardBray, any particular package you're concerned about, The issue linked to this MR might give some clarification #333

Reading the issue the code makes much more sense now. I was worried things were being moved around for preference but I wasn't aware unused packages were being installed and there was a desire to build a production only image of the project.

**Pull request merged**